ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -167,6 +109,36 @@ const REFERENCE_LOOKUP = ['37', '38'].reduce((acc, genome) => ({
   },
 }), {})
 
+const GTEX_TRACK_OPTIONS = GTEX_TRACKS.map((track, i) => ({
```

Comment:
just compute and export the constants you need from the constants file, don't put half the constants there and half of them here